docs/0.3/purchase_order_family_specification.md

```diff
@@ -186,49 +194,58 @@ message PurchaseOrderPayload {
   enum Action {
     UNSET_ACTION = 0;
     CREATE_PO = 1;
-    CREATE_VERSION = 2;
-    UPDATE_VERSION = 3;
+    UPDATE_PO = 2;
+    CREATE_VERSION = 3;
+    UPDATE_VERSION = 4;
   }
-
   Action action = 1;
-  string org_id = 2;
-  string public_key = 3;
   // Approximately when the transaction was submitted, as a Unix UTC timestamp
-  uint64 timestamp = 4;
+  uint64 timestamp = 3;
```

Comment:
These protobuf msgs were updated so the numbering is correct- this ex should be updated too

Reply:
updated